charts/tekton-apps/templates/pvc.yaml

```diff
@@ -9,8 +9,13 @@
                   "namespace"         $.Release.Namespace
                   "storageClassName"  $.Values.storageClassName }}
 
+{{- /* initialize `argocd` variable with either `$component.argocd` dict if it exists or with empty dict */}}
+{{ $argocd := ternary $component.argocd dict (hasKey $component "argocd") -}}
```

Comment:
I don't like it - why do we need additional field? We can use `component.name`== `wordpress` to understand what we're dealing with.

Reply:
done